### DISCUSSION TITLE
personal-development-ordibehesht-1402/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#13] [Announcements]
# personal-development-ordibehesht-1402/

گزارش توسعه مهارتی من در اردیبهشت ۱۴۰۲.

https://aprd.ir/personal-development-ordibehesht-1402/

**Comment by HashemAlavi** [comment]
سلام امیر پورمند عزیز. شبت بخیر:)

ممنون از گزارشی که نوشتی. خیلی کامل بود و حس قشنگی گرفتم ازش.

ولی یک تیکه ترسناک هم دارم می‌بینم و اونم چیزی که در مورد استرس نوشتی. گشتن به دنبال خونه و رفتن به مصاحبه کاری و همزمان هم درگیر پایان نامه و موضوعات مختلف بودن به‌نظر خیلی سخت و طاقت‌فرسا میاد. دارم آینده خودم رو تصور می‌کنم و اینکه هندل کردن اینا با هم به صورت همزمان چقدر می‌تونه سخت بشه. فکر کنم منم زودتر باید برم سراغ مدیریت استرس قبل از وارد شدن استرس‌های این مدلی به زندگیم که آمادگیشو داشته باشم.😂

امیدوارم که به خوبی از پس همه این چالش‌ها بر بیای. خسته نباشی:)

**Reply by pourmand1376** [reply]
سلام آقا هاشم. 

 کم کم دارم میفهمم یه مهارت هایی هست که مهم تر از مهارتهای فنی هست. همین مهارت هندل کردن استرس خیلی چیز مهمی هست. اتفاقا تاثیرگذارتر هم هست از نظر شخصی. احتمالا آنهایی که گفتند «بسیار سفر باید تا پخته شود خامی» هم به استرس اشاره داشتند چون تو زندگی عادی که استرسی نیست. تو تغییر استرس هست. 

البته استرس دائمی آدم رو نابود می‌کنه. ولی فکر کنم کمش خوبه.

**Comment by mahshidgh96** [comment]
سلام امیرجان
من کامنت شما رو زیر بحث chatgpt دیده بودم و الان هم از لینک های روزنوشته ها اومدم اینجا و گزارش توسعه مهارتی شما رو خوندم
خیلی جالب بود من همیشه دلم میخواست که این هارو جایی بنویسم اما اهمال کاری مانع میشد
الان که دیدم چقدر ساده و روان نوشتی، محرکی شد که شاید منم شروع کنم به نوشتن
راستی اگه هنوز داری مصاحبه میری من میتونم تپسی رو هم بهت پیشنهاد کنم 
اگه خواستی میتونم referral  رزومه ات رو به منابع انسانی ارجاع بدم
راستی فکر کنم برای سربازی هم بتونن کمک کنن
حتمن خوشحال میشن اگه رزومه ات رو ببینن
من امروز لینکدینت رو دیدم و فهمیدم که هم یزدی هستی هم خفن و پرتلاش
من دوستای یزدی خفن کم نداشتم
زنده باد
سلامت و پیشرو باشی

**Reply by pourmand1376** [reply]
سلام مهشید خانم

شما لطف دارید. راستش تپسی یکی از اولین جاهایی بود که اپلای کردم و خب خبری نشد. منم گفتم لابد پوزیشن خالی برای من ندارند و بیشتر پیگیری نکردم.

الان با یک شرکت دیگه مصاحبه کردم و بهشون جواب قطعی دادم و در پروسه استخدام هستم. ولی اگر مجددا دنبال کار بودم، حتما بهتون پیام میدم. 

راجع به نوشتن هم فکر می‌کنم اگر بتونید به اینرسی اولیه غلبه کنید و بنویسید، رو غلتک می‌افتید و دیگه ولش نمی‌کنید. 

خودمم نسبت به نوشتن حس خوبی دارم خصوصا این که با افراد جدید آشنا میشم و هم‌فکر پیدا می‌کنم، جالبه. 

امیدوارم که به زودی شما هم یک وبلاگ راه بندازید و لینک‌اش رو برام بذارید که بیام مطالعه کنم. 

با مهر
امیر
